perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -427,34 +427,45 @@ sub open311_get_update_munging {
         return;
     }
 
+    # Fetch outgoing notes
+    my $event = $request->{echo_event} || do {
+        my $echo = $self->feature('echo');
+        $echo = Integrations::Echo->new(%$echo);
+        my $event = $echo->GetEvent($request->{service_request_id}); # From the event, not the report
+        $echo->log($event->{Data});
+        $event;
+    };
+    my $data = Integrations::Echo::force_arrayref($event->{Data}, 'ExtensibleDatum');
+    my $notes = "";
+    foreach (@$data) {
+        $notes = $_->{Value} if $_->{DatatypeName} eq 'Veolia Notes';
+    }
+
     # An update from Echo with resolution code 1252
     my $code = $comment->get_extra_metadata('external_status_code') || '';
     if ($code eq '1252') {
```

Comment:
I don't know if it would be good to have a comment or constant to explain what code 1252 refers to exactly.